src/main/webapp/scripts/popular.js

```diff
@@ -1,14 +1,20 @@
+import { getTrendingFromYoutubeApi } from './list-video-service.js';
+
 function loadPopular() {
-  loadApi(function() {
-    getTrendingFromYoutubeApi()
-    .then((response) => {
-      for(item of response.items) {
-        // Ready in a later PR: display the item nicely on the page
-        console.log(item);
-      }
-    })
-    .catch((error) => {
-      updateDom(error.message, 'popular-list-container');
-    });
+  return loadApi().then(() => popular.getTrendingFromYoutubeApi()).then((response) => {
```

Comment:
Nt: Is `popular` needed here?

Reply:
yes it is, because of how the module is exported (it's kinda weird I know but I couldn't get modules working with sinon tests another way)